models/articles.model.js

```diff
@@ -10,4 +10,32 @@ exports.getArticleById = (id) => {
         }
         return rows[0]
     })
-}
\ No newline at end of file
+}
+
+exports.getArticles = () => {
+    return db
+      .query(
+        `SELECT articles.author,articles.title,articles.article_id,articles.topic,
+      articles.created_at,articles.votes,articles.article_img_url,CAST(COUNT(comments.comment_id)AS INT) AS comment_count
+  FROM articles
+  LEFT JOIN comments ON comments.article_id = articles.article_id
+  GROUP BY 
+      articles.article_id
+  ORDER BY articles.created_at DESC;`
+      )
+      .then(({ rows }) => {
+        return rows;
+      });
+  };
+
+  exports.selectArticleComments = (id) => {
+    return db.query(`SELECT * FROM comments WHERE article_id = $1`, [id]).then(({rows}) => {

```

Comment:
The ticket asks:

Comments should be served with the most recent comments first.